security/pkg/util/jwtutil.go

```diff
@@ -25,27 +25,41 @@ import (
 
 // IsJwtExpired checks if the JWT token is expired compared with the given time, without validating it.
 func IsJwtExpired(token string, now time.Time) (bool, error) {
+	expiration, err := GetExp(token)
+	if expiration.IsZero() {
+		if err != nil {
+			return true, err
+		}
+		return false, nil
```

Comment:
Shall we return true in this case? Can expiration be zero?

Reply:
Returning true breaks the original logic.
expiration cannot be zero unless it is not set, which means this token is always valid.
zero value time is `The zero value of type Time is January 1, year 1, 00:00:00.000000000 UTC`, official explanation is here https://golang.org/pkg/time/#Time

Reply:
The code change in this file is a code refactoring. I don't want to change the behavior of `IsJwtExpired`. If we want to change this, we should make sure that change does not break any existing flows, I would prefer to make changes in a separate PR.